systemtest/src/test/java/io/strimzi/systemtest/metrics/PrometheusST.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.metrics;
+
+import io.strimzi.systemtest.AbstractST;
+import io.strimzi.systemtest.utils.StUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static io.strimzi.systemtest.Constants.REGRESSION;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@Tag(REGRESSION)
+public class PrometheusST extends AbstractST {
+
+    private static final Logger LOGGER = LogManager.getLogger(PrometheusST.class);
+
+    public static final String NAMESPACE = "prometheus-test";
+
+    private static final String PROMETHEUS = "prometheus";
+    private static final String PROMETHEUS_POD = "prometheus-prometheus-0";
+    private static final String ALERTMANAGER = "alertmanager";
+    private static final String ALERTMANAGER_POD = "alertmanager-alertmanager-0";
+
+    @Test
+    public void testPrometheusService() {
+        assertThat("Prometheus service not found", kubeClient().getService(PROMETHEUS) != null);
```

Comment:
Out of curiosity ... why are we now using `assertThat`? the syntax seems to tally weird because you read it as _Assert that prometheus service is not found_ when the expectations are actually different.

Reply:
Message is optional, it's an error message which is showed when assert fails. I am kinda used to use `assertThat` instead of other asserts, not sure if I have some specific reason.